File: src/app/browser/main/tab/label/title.cpp
#include "title.hpp"

using namespace app::browser::main::tab::label;

Title::Title()
{
    set_text(
        _("New page")
    );

    set_ellipsize(
        Pango::EllipsizeMode::END
    );

    set_width_chars(
        WIDTH_CHARS
    );

    set_single_line_mode(
        true
    );
}

File: src/app/browser/main/tab/label/title.rs
use gtk::Label;

pub fn new() -> Label
{
    return Label::builder()

        .label(
            "New page"
        )

        .ellipsize(
            gtk::pango::EllipsizeMode::End
        )

        .width_chars(
            16
        )

        .single_line_mode(
            true
        )

        .build();
}